client/src/Chat.jsx

```diff
@@ -47,37 +55,34 @@ const Messages = ({ user }) => {
         <div
           key={id}
           style={{
-            display: "flex",
-            justifyContent: user === messageUser ? "flex-end" : "flex-start",
-            paddingBottom: "1em",
-          }}
-        >
+            display: 'flex',
+            justifyContent: user === messageUser ? 'flex-end' : 'flex-start',
+            paddingBottom: '1em'
+          }}>
           {user !== messageUser && (
             <div
               style={{
                 height: 50,
                 width: 50,
-                marginRight: "0.5em",
-                border: "2px solid #e5e6ea",
+                marginRight: '0.5em',
+                border: '2px solid #e5e6ea',
                 borderRadius: 25,
-                textAlign: "center",
-                fontSize: "18pt",
-                paddingTop: 5,
-              }}
-            >
+                textAlign: 'center',
+                fontSize: '18pt',
+                paddingTop: 5
+              }}>
               {messageUser.slice(0, 2).toUpperCase()}
             </div>
           )}
 
           <div
             style={{
-              background: user === messageUser ? "#58bf56" : "#e5e6ea",
-              color: user === messageUser ? "white" : "#black",
-              padding: "1em",
-              borderRadius: "1em",
-              maxWidth: "60%",
-            }}
-          >
+              background: user === messageUser ? '#58bf56' : '#e5e6ea',
+              color: user === messageUser ? 'white' : '#black',
```

Comment:
With so much inner styling, you can move them to css file and leave here only styles, which are calculated programmatically.
If the purpose of this project is to learn using WebSockets & graphql, then it's fine, no problems.

Reply:
The purpose of this project is to learn websocket and graphql. Not really a frontend person